docs/index.rst

```diff
@@ -50,23 +50,28 @@ Useful links:
 
 .. toctree::
    :maxdepth: 2
-   :caption: Layout and Panels
+   :caption: Introduction
 
    layout
+   security
+   remix_tutorials_learneth
+   locations
+   FAQ
 
 .. toctree::
    :maxdepth: 2
-   :caption: Tour of default modules
+   :caption: Core Modules
 
    file_explorer
    plugin_manager
    settings
    solidity_editor
    terminal
+   remixd
 
 .. toctree::
    :maxdepth: 2
-   :caption: Tour of typical solidity modules
+   :caption: Solidity modules
```

Comment:
`m` in modules should be capital

Reply:
true -but the capitalization is done in the readthedocs CSS

Reply:
ok